_posts/2017-05-14-tiler.md

```diff
@@ -18,19 +16,19 @@ One thing we found when exploring how we might take Ordnance Survey Open Data su
 
 Thankfully we found lots of tools that did individual pieces of what we needed; we just needed to package them together. This was when the idea of [Tiler](https://github.com/geovation/tiler) was born. Tiler aims to be a (relatively) simple to use pipeline for taking traditional vector data from sources like Shapefiles, GeoJSON, GML or PostGIS and turning them into raw Vector Tiles that you can use with clients such as Mapbox GL or OpenLayers (as examples).
```

Comment:
Add link to OpenLayers.  In previous paragraphs, change 'restyling' to 're-styling' and 'some what' to 'somewhat'